subprojects/reporting/src/integTest/groovy/org/gradle/api/reporting/plugins/BuildDashboardPluginIntegrationTest.groovy

```diff
@@ -260,6 +260,7 @@ class BuildDashboardPluginIntegrationTest extends WellBehavedPluginTest {
     }
 
     @ToBeFixedForInstantExecution
+    @Requires(TestPrecondition.JDK13_OR_EARLIER) // Current CodeNarc does not work with JDK14 because of the groovy version it uses
     void 'enabling an additional report renders buildDashboard out-of-date'() {
```

Comment:
We could force the version of Groovy to be one that is supported by the JDK we're using.  I don't think it's important to use the version of Groovy that CodeNarc declares for these tests (we just want some reports).